Documentation/compiling-seissol.rst

```diff
@@ -1,44 +1,96 @@
-.. _Compiling_and_running_seissol:
+.. _build_seissol:
 
-Compiling and running SeisSol
+Compiling and Running SeisSol
 -----------------------------
 
 Compiling SeisSol
 ~~~~~~~~~~~~~~~~~
-Get the latest version of SeisSol on git by cloning the whole repository
+Get the latest version of SeisSol on git by cloning the whole repository,
 including all submodules:
 
 .. code-block:: bash
 
    git clone --recursive https://github.com/SeisSol/SeisSol.git
 
-If you have compiled :ref:`seissol-env with spack<spack_installation>`, load the module and compile SeisSol with (e.g.):
+Then, given that you have installed all required dependencies, you may compile SeisSol for CPUs as follows (or use ``ccmake`` for a GUI):
 
 .. code-block:: bash
 
-   mkdir build-release && cd build-release
-   CC=mpicc CXX=mpiCC FC=mpif90 cmake -DCOMMTHREAD=ON -DNUMA_AWARE_PINNING=ON -DASAGI=ON -DCMAKE_BUILD_TYPE=Release -DHOST_ARCH=skx -DPRECISION=double -DORDER=4 -DGEMM_TOOLS_LIST=LIBXSMM,PSpaMM ..
+   mkdir build && cd build
+   cmake -DCMAKE_BUILD_TYPE=Release -DHOST_ARCH=hsw -DPRECISION=double -DORDER=4 -DGEMM_TOOLS_LIST=Eigen -DEQUATIONS=elastic -DGRAPH_PARTITIONING_LIBS=none ..
```

Comment:
Given the fact that a fair amount of people will just copy-paste this line, should we use `hsw`, `libxsmm`, `parmetis` as default?

Reply:
Also, why not `GEMM_TOOLS_LIST=auto`?

Reply:
I'd vote for simplicity, which is to provide one set of instructions, which works for almost everybody with good performance. 
Installing libxsmm is easy, every user should do it sooner or later anyways. If they've compiled netcdf/hdf5 successfully, they will also easily install libxsmm. Thus, I don't see a reason for distinguishing between minimal/optimal, but I would just give a default.